app/adachi/todo/page.tsx

```diff
@@ -0,0 +1,100 @@
+'use client'
+import { useState } from 'react'
+
+interface AppProps {
+  initialText?: string
+}
+
+const App = ({ initialText = '' }: AppProps): JSX.Element => {
+  const [text, setText] = useState<string>(initialText)
+  const [tasks, setTasks] = useState<string[]>([])
+  const [isInputVisible, setIsInputVisible] = useState<boolean>(false)
+  const [error, setError] = useState<string>('')
+
+  const handleButtonClick = () => {
+    setIsInputVisible(true)
+    setError('') // エラーをクリア
+  }
+
+  const handleAddTask = () => {
+    if (text.trim() === '') {
+      setError('タスクを入力してください')
+      return
+    }
+    setTasks([...tasks, text]) // 新しいタスクを追加
+    setText('') // タスク追加後に入力欄をクリア
+    setIsInputVisible(false) // 入力欄を非表示にする
+    setError('') // エラーをクリア
+  }
+
+  const handleDeleteTask = (index: number) => {
+    const newTasks = tasks.filter((_, i) => i !== index) // 指定されたインデックスのタスクを削除
+    setTasks(newTasks)
+  }
+
+  const handleCancel = () => {
+    setIsInputVisible(false)
+    setText('')
+    setError('')
+  }
+
+  return (
+    <div className="App flex h-80 flex-col items-center justify-center font-mono text-lg">
+      <h1 className="m-8 text-4xl font-bold">TODO List</h1>
+      {!isInputVisible && (
+        <button
+          onClick={handleButtonClick}
+          className="rounded-full bg-green-600 px-10 text-xl text-white"
+        >
+          ＋
+        </button>
+      )}
+      {isInputVisible && (
+        <div className="text-center">
+          <input
+            value={text}
+            onChange={(event) => setText(event.target.value)}
+            className="rounded-md bg-gray-100 ring-2 ring-black"
+          />
+          <br />
+          {error && (
+            <div style={{ color: 'red' }} className="m-1 text-center">
+              {error}
+            </div>
```

Comment:
`css`は、`tailwind`で統一した方が綺麗だと思います。
```suggestion
            <div className="m-1 text-center text-red-600">
              {error}
            </div>
```